src/components/CountrySelector/index.js

```diff
@@ -44,16 +44,18 @@ const getCountryOptionTemplate = (country: CountryData) =>
     <span class="${style.countryLabel}">${country.name}</span>`
 
 class CountrySelection extends Component<Props, State> {
+  autocompleteEl: ?HTMLDivElement
   state = {
     showNoResultsError: false,
   }
 
-  handleCountrySearchConfirm = (selectedCountry: CountryData) => {
+  handleCountrySearchConfirm = async (selectedCountry: CountryData) => {
     if (selectedCountry) {
       this.setState({
         showNoResultsError: false,
       })
-      this.props.actions.setIdDocumentIssuingCountry(selectedCountry)
+      await this.props.actions.setIdDocumentIssuingCountry(selectedCountry)
+      this.autocompleteEl.elementReferences[-1].blur()
```

Comment:
What/why does `elementReferences` do & need to be -1?

Reply:
That's the reference of[ the input element ](https://github.com/alphagov/accessible-autocomplete/blob/a148b00c85118361071ec9ecc28a1dc6d905d6c7/src/autocomplete.js#L366). I agree, this is not very readable. I will just get the element by id.